internals/daemon/api.go

```diff
@@ -94,6 +94,15 @@ var API = []*Command{{
 	Path:   "/v1/checks",
 	UserOK: true,
 	GET:    v1GetChecks,
+}, {
+	Path:   "/v1/notices",
+	UserOK: true,
```

Comment:
Per voice discussion, use use `UserOK` on all existing endpoints, so we'll leave as is for now. We can later merge in what snapd agrees on if appropriate.

Reply:
Just a side comment on this: I think the user story also comes into play if we allow the client to connect over TCP sockets (remote connections)? KernOS client would really benefit from remote client connections (I think we will schedule this in the next cycle), and if I recall my previous hack, I had to comment our some user permission checks. Just a heads-up that if this topic is touched, I would be happy to participate as a stakeholder.

Reply:
In yesterday's call with snapd, we agree that the implementation with user support will be done by the snapd team, but first on Pebble, so that both of you can also easily participate in the process.